core/cmd/app.go

```diff
@@ -184,6 +184,11 @@ func NewApp(client *Client) *cli.App {
 							Name:  "password, p",
 							Usage: "text file holding the password for the node's account",
 						},
+						cli.Int64Flag{
+							Name:  "ignore-before-height, i",
+							Usage: "historical block height before which to ignore log-initiated jobs",
+							Value: -1,
+						},
```

Comment:
@se3000 @spooktheducks I may have missed out of the conversation, but what do you think about a positive argument name?  the ticket suggested '--replay-from-block'. The difference being "I want these" and "I want everything but these", the prior is simpler for me personally.  Not a blocker, feel free to disagree :) though I suggest in the future adding the rationale behind these decisions (its different from the ticket) in the PR comment and / or commit comment.

Reply:
:+1: